CMakeLists.txt

```diff
@@ -34,8 +34,7 @@ include(LXQtCompilerSettings NO_POLICY_SCOPE)
 include(LXQtTranslate)
 
 if(${CMAKE_SYSTEM_NAME} MATCHES "OpenBSD" OR ${CMAKE_SYSTEM_NAME} MATCHES "NetBSD")
-    message(WARNING "${CMAKE_SYSTEM_NAME} is not supported by lxqt-admin-user")
-    message(WARNING "${CMAKE_SYSTEM_NAME} is not supported by lxqt-admin-time")
+    add_subdirectory(lxqt-admin-user)
```

Comment:
What's the solution?

Reply:
Wouldn't this work?
```
if(${CMAKE_SYSTEM_NAME} MATCHES "OpenBSD" OR ${CMAKE_SYSTEM_NAME} MATCHES "NetBSD")
    add_subdirectory(lxqt-admin-user)
else()
    add_subdirectory(lxqt-admin-user)
    add_subdirectory(lxqt-admin-time)
endif()

```
lxqt-admin-user works just fine. Its only lxqt-admin-time that causes problems due to systemd dependency.

Reply:
> lxqt-admin-user works just fine. Its only lxqt-admin-time that causes problems due to systemd dependency.

Well then
1. the commit message is misleading; please, change it to not make confusion
2. please, leave the warning about not supported `lxqt-admin-time`